store-implementation/federated-store/src/main/java/uk/gov/gchq/gaffer/federatedstore/FederatedStoreCacheTransient.java

```diff
@@ -40,10 +41,14 @@ public FederatedStoreCacheTransient() {
     }
 
     public FederatedStoreCacheTransient(final String cacheNameSuffix) {
-        super(String.format("%s%s", CACHE_SERVICE_NAME_PREFIX,
+        super(getCacheNameFrom(cacheNameSuffix));
+    }
+
+    public static String getCacheNameFrom(final String cacheNameSuffix) {
+        return String.format("%s%s", CACHE_SERVICE_NAME_PREFIX,
                 nonNull(cacheNameSuffix)
-                        ? "_" + cacheNameSuffix.toLowerCase()
-                        : ""));
+                        ? "_" + cacheNameSuffix.toLowerCase(Locale.UK)
```

Comment:
Is this needed?

Reply:
changed to locale.default() to match rest of gaffer.